### PR TITLE
CF: adding redirect, further links on get started page

### PR DESCRIPTION
**Pull request by mgamini**
tenor-69474407

Signed-off-by: Garrett Amini <kadjar@gmail.com>

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/get-started/cloudfoundry/index.html.slim [hunk 1]
```diff
@@ -20,20 +20,32 @@ div.get-started--container
...
+              = link_to 'Chef Habitat Addresses the Complexities of Moving Containers into Production', 'https://thenewstack.io/chef-habitat-addresses-issues-moving-containers-production/'
+            li.link-list--item
+              = link_to 'Chef cooks up a new Habitat for packaging apps', 'https://www.cbronline.com/enterprise-it/chef-cooks-new-habitat-packaging-apps/'
             li.link-list--item
-              = link_to 'Cloud Foundry Docs', 'https://www.cloudfoundry.org/docs-download/'
+              = link_to 'Using Habitat to Build the Industrial Cloud at GE Digital', 'https://blog.chef.io/2017/10/12/using-habitat-to-build-the-industrial-cloud-at-ge-digital/'
             li.link-list--item
-              = link_to 'Habitat Docs', '/docs/'            
\ No newline at end of file
+              = link_to 'How Habitat can Boost your Chef Ecosystem', 'https://www.youtube.com/watch?v=F1L3RlNnJqI'
```
Comment (cnunciato) [inline www/source/get-started/cloudfoundry/index.html.slim]: Title-case would make this `How Habitat Can Boost your Chef Ecosystem `

File: www/config.rb [hunk 2]
```diff
@@ -228,3 +228,4 @@ def render_markdown(text)
 redirect 'demo/build-system/index.html', to: '/demo/build-system/steps/1'
 redirect 'demo/process-supervisor/index.html', to: '/demo/process-supervisor/steps/1'
 redirect 'legal/index.html', to: '/legal/licensing'
+redirect 'cloudfoundry', to: '/get-started/cloudfoundry'
```
Comment (cnunciato) [inline www/config.rb]: I think this needs to be `redirect 'cloudfoundry/index.html', to: '/get-started/cloudfoundry' `. You can verify these locally by building and then separately serving the built site:

```
make build
cd build
php -S localhost:3000
```

and browse to http://localhost:3000/cloudfoundry

File: www/source/get-started/cloudfoundry/index.html.slim (hunk 1 above)
Comment (cnunciato) [reply www/source/get-started/cloudfoundry/index.html.slim]: Or `How Habitat can boost your Chef ecosystem` would work, too, but the YouTube page just has it wrong. :smile:

**Comment by cnunciato** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/353045546).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/353045546).

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/353045546).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/353045546).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

**Pull request merged**